lib/patcher.js

```diff
@@ -340,7 +340,7 @@ function findChildReprints(newPath, oldPath, reprints) {
         if (k === "loc")
             continue;
 
-        if (!findAnyReprints(newPath.get(k), oldPath.get(k), reprints))
+        if (!findAnyReprints(newPath.get(k), oldPath.get(k), reprints, options))
```

Comment:
I might be crazy, but I can't tell where `options` is used in this file. It seems like it's just being passed through the various `find*` functions but never examined. Am I missing something obvious?